app/views/trains/_form.html.slim

```diff
@@ -0,0 +1,28 @@
+= form_for :train, method: :patch, url: train_path, local: true do |f|
```

Comment:
with `:patch` being hardcoded here will this work for post?
ideally dont want to be ignoring something that we are going to have to fix next PR

Reply:
this still isnt generic, but im guessing you have a plan for this when doing the create form?